twoopstracker/authentication/urls.py

```diff
@@ -1,10 +1,17 @@
+from dj_rest_auth.registration.views import ConfirmEmailView, VerifyEmailView
 from django.conf.urls import include
 from django.urls import path
 
 from .views import GoogleLogin
 
 urlpatterns = [
+    path(
+        "account-confirm-email/",
+        VerifyEmailView.as_view(),
+        name="account_email_verification_sent",
+    ),
     path("google/", GoogleLogin.as_view(), name="google_login"),
+    path("registration/account-confirm-email/<str:key>/", ConfirmEmailView.as_view()),
```

Comment:
Do we need this? As in clicking the link in verification email should take you to the _front-end_ and the front-end should be the one extracting the key and posting it to `/registration/verify-email/` API end-point.

Reply:
yes, that's sound right.